src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -35,9 +34,19 @@ trait ClassLikeSupport:
       signatureOnly: Boolean = false,
       modifiers: Seq[Modifier] = classDef.symbol.getExtraModifiers(),
     ): DClass = 
-      val supertypes = getSupertypes(classDef).map{ case (symbol, tpe) =>
-        LinkToType(tpe.dokkaType.asSignature, symbol.dri, kindForClasslike(symbol))
+      def hackGetParents(classDef: ClassDef): Option[List[Tree]] = scala.util.Try(classDef.parents).toOption
```

Comment:
Why is a `Try` necessary here? My intuition would be that by using `Try` here, we are obscuring errors that we should know about.

Reply:
This `Try` here is because of problem that code compiles, but at runtime fails claiming https://pastebin.pl/view/41317bda. @romanowski  identified it as problem with Tasty and advised me to wrap it with `hack` function, but maybe you could bring some more insdight. Here are listed `LinkToTypes` that cannot have calculated parent becuase of that error. https://pastebin.pl/view/b06d09bb

Reply:
Ah, I see. This does look like a bug in Tasty. A comment explaining that would be a good idea. We'll try to get issues like that fixed soon after repo merge.